extensions/cli/landsat8/src/main/java/mil/nga/giat/geowave/format/landsat8/Landsat8AnalyzeCommand.java

```diff
@@ -18,10 +18,10 @@
 import mil.nga.giat.geowave.core.cli.api.DefaultOperation;
 import mil.nga.giat.geowave.core.cli.api.OperationParams;
 
-@GeowaveOperation(name = "analyze", parentOperation = Landsat8Section.class)
+@GeowaveOperation(name = "analyze", parentOperation = Landsat8Section.class, restEnabled = GeowaveOperation.RestEnabledType.POST)
 @Parameters(commandDescription = "Print out basic aggregate statistics for available Landsat 8 imagery")
 public class Landsat8AnalyzeCommand extends
-		DefaultOperation implements
+		DefaultOperation<Void> implements
```

Comment:
for this to actually do anything it needs to print something out in CLI and return something in services (thats really the only outcome of "analyze").  This is true of the entire landsat8 geowave project (although at least other command do have more outcomes, but they should all print analysis results).